packages/app/src/vis-packs/utils.ts

```diff
@@ -0,0 +1,36 @@
+import type { Entity } from '@h5web/shared';
+
+import { getSupportedVis as getSupportedCoreVis } from './core/pack-utils';
+import type { VisDef } from './models';
+import {
+  getNxDefaultPath,
+  getSupportedVis as getSupportedNxVis,
+} from './nexus/pack-utils';
```

Comment:
As we no longer have a `visPack` concept, it should not be kept in the structure of the files

Reply:
I'll investigate this after I've renamed `VisPackResolver` and `Visualizer` and merged the PR :+1: